protected/humhub/libs/Html.php

```diff
@@ -145,6 +145,7 @@ public static function backButton($url = '', $options = [])
     public static function containerLink(ContentContainerActiveRecord $container, $options = [])
     {
         $options['data-contentcontainer-id'] = $container->contentcontainer_id;
+        $options['data-guid'] = $container->guid;
```

Comment:
Maybe we should mention this in the changelog:

- Enh #4571: `humhub/libs/Html::containerLink()` now adds a  `data-guid` attribute

Reply:
Done ! But just want you to know  
This code is not in my branch
$options['data-guid'] = $container->guid;